api/MediaKeySystemConfiguration.json

```diff
@@ -15,10 +15,10 @@
             "version_added": "≤79"
           },
           "firefox": {
-            "version_added": null
+            "version_added": "43"
```

Comment:
This dictionary is used in https://developer.mozilla.org/en-US/docs/Web/API/Navigator/requestMediaKeySystemAccess#browser_compatibility so support should probably go back to 38. The original tracking bug was https://bugzilla.mozilla.org/show_bug.cgi?id=1095257 and the dictionary was called `MediaKeySystemOptions`, but the name of a dictionary isn't observable.

State of that dictionary right before it was renamed:
https://github.com/mozilla/gecko-dev/blob/0d4e98b0d28e08c62600f04035ca9ea7e607e862/dom/webidl/MediaKeySystemAccess.webidl#L19-L27